templates/github/.github/workflows/scripts/script.sh.j2

```diff
@@ -186,7 +186,12 @@ fi
 if [ -f $FUNC_TEST_SCRIPT ]; then
   source $FUNC_TEST_SCRIPT
 else
+    {% if parallel_test_workers == 0 %}
     pytest -v -r sx --color=yes --pyargs {{ plugin_snake }}.tests.functional
+    {% else -%}
+    pytest -v -r sx --color=yes --pyargs {{ plugin_snake }}.tests.functional -m parallel -n {{ parallel_test_workers }}
```

Comment:
you need to configure that marker for pytest to avoid warnings

Reply:
I agree, but I don't think the plugin template can append that config to pyproject.toml can it? It would be great if it could.

Reply:
Actually I was able to do this here:  https://github.com/pulp/pulp-smash/pull/1268/files

Reply:
At least for new bootstraps it should. Also this is kind of a reference to update those files.

And in general, yes there is room for improvement. e.g. it's a pain to update the docs machinery from the template. The CI parts is really the best maintained part.

Reply:
I see them moved to pulp-smash.